azbus/receiver.go

```diff
@@ -87,6 +87,8 @@ type Receiver struct {
 	receiver *azservicebus.Receiver
 	options  *azservicebus.ReceiverOptions
 	handlers []Handler
+	ctx      context.Context
```

Comment:
Please can you add a comment stating why it is OK to stash a Context in a struct (a wise man once told me that was an anti-pattern)

Reply:
removed context from struct